core/src/main/java/io/forestframework/core/http/routing/PlainHttpRoutingMatchResult.java

```diff
@@ -42,8 +42,8 @@ public void select(HttpServerRequest request,
 
     public void addResult(HttpServerRequest request, List<Routing> routings, Map<String, String> pathVariables) {
         routings.stream()
-                .collect(Collectors.groupingBy(Routing::getType))
-                .forEach((key, value) -> putIntoResult(request, key, value, pathVariables));
+            .collect(Collectors.groupingBy(Routing::getType))
```

Comment:
Can you also add rules to `editorconfig` so we don't have these space changes in the future?

Reply:
Sure!